File: BubbleSort/BubbleSort.java
import java.util.*;
public class BubbleSort
{
    public static void main(String[] args)
    {
        Scanner input = new Scanner(System.in);
        int n = Integer.parseInt(input.nextLine());
        int[] theArray = new int[n];
        for(int i = 0; i < n; i++)
            theArray[i] = input.nextInt();
        Bubblesort(theArray);
        for (int i = 0; i < n; i++)
            System.out.print(theArray[i]+" ");
        System.out.println();
        input.close();
    }
    public static void Bubblesort(int[] theArray)
    {
        for(int i = 1; i < theArray.length; i++)
        {
            for (int j = 0; j < theArray.length-1; j++)
            {
                if(theArray[j] > theArray[j+1])
                {
                    int tmp = theArray[j];
                    theArray[j] = theArray[j+1];
                    theArray[j+1] = tmp;
                }
            }
        }
    }
}

File: BubbleSort/BubbleSort.py
def Bubblesort(theArray):
    for i in range(1,n):
        for j in range(0,n-1):
            if theArray[j] > theArray[j+1]:
                tmp = theArray[j]
                theArray[j] = theArray[j+1]
                theArray[j+1] = tmp

n = int(input())
theArray = input().split(" ")
for x in theArray:
    x = int(x)
Bubblesort(theArray)
for x in theArray:
    print(x, end = " ")
print()
